Makefile.common.mk

```diff
@@ -19,13 +19,38 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-IMG = gcr.io/istio-testing/build-tools:2019-08-05
+IMG = docker.io/sdake/build-tools:2019-08-06
```

Comment:
Do you really want to use the image from your repo? Just not changed after testing?

Reply:
@geeknoid indicated he would push the image this morning.  Unfortunately it wasn't pushed and I don't desire the repo to be blocked all day.

We can follow this up with a PR to change it to the official image once @geeknoid is back in the office.